.github/workflows/ci_backend.yml

```diff
@@ -0,0 +1,52 @@
+name: Django CI
+
+on: [push]
+
+defaults:
+  run:
+    working-directory: ./backend
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:12-alpine
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: backend
+        ports:
+          - 5432:5432
+        # needed because the postgres container does not provide a healthcheck
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+
+
+    strategy:
+      matrix:
+        python-version: [3.6]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install dependencies
+      run: |
+        pip install pipenv
+        pipenv install --dev
+    - name: Install node dependencies (mjml etc)
+      run: npm install
+      working-directory: ./
+    - name: Run tests
+      run: |
+          pipenv run "pytest -x"
+      env:
+        CI: true
+        DATABASE_URL: "postgres://postgres@localhost:5432/backend"
```

Comment:
I _do_ not understand what I'm doing wrong to make django not see the postgres server here…